### ISSUE TITLE
Check .js files with ESLint

### ISSUE THREAD

**Issue by EvanHahn** [helmetjs/helmet#349]
Currently, ESLint only checks `.ts` files:

https://github.com/helmetjs/helmet/blob/de66579e4e43c0239c3caae37b5005b0eece4df8/package.json#L47

However, it should also check the small number of `.js` files in the project. (None of these files are published.) When we're all done, the command should probably look like this:

```json
 "lint:eslint": "eslint .", 
```

Because ESLint is currently configured for TypeScript, we may need to do some overrides.

**Comment by njmulsqb** [comment]
Hi, I am willing to take on this issue, I am a beginner and wants to contribute to FOSS.

Can you assign me this task? I might ask some questions on this issue while solving this issue.

Thanks

**Comment by EvanHahn** [comment]
Sure! I've just assigned it to you.

Feel free to ask questions here or email me@evanhahn.com.

**Comment by njmulsqb** [comment]
Well, that was quicker than I expected.

I cloned the project and ran `npm run lint` which gave absolutely no lint issues, then I modified the `package.json` file to include JS files while linting the project and then again ran `npm run lint`, I was expecting that it will return some lint issues which I will need to resolve but it also didn't show anything
Then I tried to scan the `bin` folder using eslint CLI and it also didn't show anything. It seems as if we're good to go, hence I have created the PR.
If I understood it wrong, do let me know!
